tanner/migration_script.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+import json
+import aioredis
+import psycopg2
+from tanner import redis_client, postgres_client, dbutils
+
+
+async def check_session_data(result):
+    Integers = ["zip_code", "errors", "accepted_paths", "port", "atbr", "rps", "hidden_links"]
```

Comment:
in raw data, there s no atbr and rps,
rps -->requests_in_second
atbr --> approx_time_between_requests